tailscale/DOCS.md

```diff
@@ -147,6 +148,31 @@ More information: [Subnet routers][tailscale_info_subnets]
 When not set, the add-on by default will advertise routes to your subnets on all
 supported interfaces.
 
+### Option: `forward_to_host`
+
+Forward incoming tailnet connections to the host's primary interface when
+userspace networking is disabled.
+
+When not set, this option is enabled by default.
+
+When userspace networking is enabled, Tailscale automatically forwards incoming
+tailnet connections to localhost.
+
+When userspace networking is disabled, the add-on can forward incoming tailnet
+connections to Home Assistant's primary host interface. This means you don't
+have to enable subnet routing just to access services on the host from the
+tailnet.
+
+**Note:** Without forwarding, services running only on the interfaces managed by
+Home Assistant (i.e. not on all interfaces), are inaccessible from the tailnet
+when userspace networking is disabled.
+
+**Note:** Tailscale's serve and funnel features have priority over this plain
+port forwarding, those connections won't be forwarded directly to the host.
+
+**Note:** Hairpinning is not implemented, do not test forwarding by accessing
+the host, from itself, through the tailscale0 interface.
```

Comment:
Having needing 3 notes here seems like this is a bad idea?

This is all things users shouldn't have to worry about?

Reply:
Maybe we need to remove the whole option and just always do it?

If you didn't notice yet: I hate options. We should remove as many as possible and just do what works for 99% of the users. Adding more options only adds more complexity and often leads to more issues and user support down the road.

../Frenck